tests/e2e/kubetest2-kops/deployer/up.go

```diff
@@ -168,15 +168,7 @@ func (d *deployer) createCluster(zones []string, adminAccess string, yes bool) e
 		"--kubernetes-version", d.KubernetesVersion,
 		"--ssh-public-key", d.SSHPublicKeyPath,
 		"--set", "cluster.spec.nodePortAccess=0.0.0.0/0",
-	}
-
-	version, err := kops.GetVersion(d.KopsBinaryPath)
-	if err != nil {
-		return err
-	}
-	if version > "1.29" {
-		// Requires https://github.com/kubernetes/kops/pull/16128
-		args = append(args, "--set", `spec.containerd.configAdditions=plugins."io.containerd.grpc.v1.cri".containerd.runtimes.test-handler.runtime_type=io.containerd.runc.v2`)
+		"--set", `spec.containerd.configAdditions=plugins."io.containerd.grpc.v1.cri".containerd.runtimes.test-handler.runtime_type=io.containerd.runc.v2`,
```

Comment:
This looks like a cleanup for handling of old kube versions that we no longer test?  (I.e. could be split into its own PR, though that would be a very short PR!)